pages/api/projects/[projectId].js

```diff
@@ -1,6 +1,9 @@
-import {CotterValidateJWT} from "cotter-node";
 import Airtable from "airtable";
+import loadStytch from '../../lib/loadStytch';
 
+const client = loadStytch();
+
+// Instantiate the Airtable client using the Sequin proxy
 const base = new Airtable({
     apiKey: process.env.AIRTABLE_API_KEY,
     endpointUrl: "https://proxy.syncinc.so/api.airtable.com",
```

Comment:
Can you change this endpoint to the `sequin` proxy → 

```suggestion
    endpointUrl: "https://proxy.sequin.io/api.airtable.com",
```